streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationIntegrationTest.java

```diff
@@ -160,30 +159,35 @@ public void shouldReduce() throws Exception {
 
         produceMessages(mockTime.milliseconds());
 
-        final List<KeyValue<String, String>> results = receiveMessages(
+        final List<KeyValueTimestamp<String, String>> results = receiveMessages(
             new StringDeserializer(),
             new StringDeserializer(),
             10);
 
         results.sort(KStreamAggregationIntegrationTest::compare);
 
-        assertThat(results, is(Arrays.asList(KeyValue.pair("A", "A"),
-            KeyValue.pair("A", "A:A"),
-            KeyValue.pair("B", "B"),
-            KeyValue.pair("B", "B:B"),
-            KeyValue.pair("C", "C"),
-            KeyValue.pair("C", "C:C"),
-            KeyValue.pair("D", "D"),
-            KeyValue.pair("D", "D:D"),
-            KeyValue.pair("E", "E"),
-            KeyValue.pair("E", "E:E"))));
+        assertThat(results, is(Arrays.asList(
```

Comment:
nit: I'm wondering if using `mockTime.milliseconds()` for each timestamp could be a source of test flakiness in the future vs. setting the result in a variable.  I dug a little into `mockTime,` and it looks like the default behavior now is not to increment.  But in the future, if that changes slightly, we could have flaky tests.  I could be a little paranoid here, and the other option is to leave this as is and deal with any changes in the future at that time.

This comment applies to other uses below as well.

Reply:
Why would it become flaky? The test should fail consistently. Hence, atm I don't see a big issue. If the behavior changes, all test using mock-time and rely ob current behavior should break (and fail consistently) and would need to be updated.